FreeTAKServer/controllers/services/API/public_API.py

```diff
@@ -120,60 +125,64 @@ def verify_token(token):
             if output:
                 return output[0].name
 
+
 @app.route("/ManagePresence")
 @auth.login_required()
 def ManagePresence():
     pass
 
+
 @app.route("/ManagePresence/postPresence", methods=[restMethods.POST])
 @auth.login_required
 def postPresence():
     try:
-        from json import dumps
-        #jsondata = {'longitude': '12.345678', 'latitude': '34.5677889', 'how': 'nonCoT', 'name': 'testing123'}
+        global APIPipe
```

Comment:
In several of these handlers `APIPipe` was not defined. I presumed that this was referencing the global variable as used elsewhere.